<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scope and Variants: Hierarchy View</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="primCount">
       <property name="text">
        <string>Total Number of Prims in Scope: %1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>USD File Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usdFilePath"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load Payloads:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="loadPayloads">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="filterLineEdit">
         <property name="minimumSize">
          <size>
           <width>355</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Filter by prim name...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findPrev">
         <property name="text">
          <string>Prev</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findNext">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findAll">
         <property name="text">
          <string>Find All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="selectPrims">
     <property name="text">
      <string>Select the prim(s) you'd like to import. Switch variants accordingly.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
